.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ros.distro": "rolling"
+}
```

Comment:
This file should not be uploaded to git as it is a config file for one specific editor.